sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -161,6 +161,8 @@ class Analyzer(
       HandleNullInputsForUDF),
     Batch("FixNullability", Once,
       FixNullability),
+    Batch("ResolveAggAliasInGroupBy", Once,
+      ResolveAggAliasInGroupBy),
```

Comment:
Sorry, not sure. Shall I check `ResolveOrdinalInOrderByAndGroupBy` works well outside the `Resolution` batch?